definitions/npm/styled-components_v2.x.x/flow_v0.53.x-/styled-components_v2.x.x.js

```diff
@@ -0,0 +1,369 @@
+// @flow
+
+type $npm$styledComponents$Interpolation = (<C: {}>(executionContext: C) => string) | string | number;
+type $npm$styledComponents$NameGenerator = (hash: number) => string;
+
+type $npm$styledComponents$TaggedTemplateLiteral<R> = {| (Array<string>, $npm$styledComponents$Interpolation): R |};
+
+// ---- FUNCTIONAL COMPONENT DEFINITIONS ----
+type $npm$styledComponents$ReactComponentFunctional<Props, DefaultProps = *> = {
+  (Props): React$Element<*>,
```

Comment:
@wtgtybhertgeghgtwtg We actually need to update this to be ``React$Node``, not ``React$Element``.